packages/account/src/Components/inline-note-with-icon/inline-note-with-icon.tsx

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Icon, Text } from '@deriv/components';
 
-const InlineNoteWithIcon = ({ icon, message, title }) => {
+type TInlineNoteWithIconExtend = {
+    icon: string;
+    message: string;
+    title: string;
+};
+
+export type TInlineNoteWithIcon = React.PropsWithChildren<TInlineNoteWithIconExtend>;
+
+const InlineNoteWithIcon = ({ icon, message, title }: TInlineNoteWithIcon) => {
```

Comment:
pls check this, we can just use TInlineNoteWithIconExtend as the prop type. no need to extend  props here